javascript/operations.js

```diff
@@ -179,16 +179,23 @@ export default {
     })
   },
 
-  callMethod: operation => {
+  invokeMethod: operation => {
     processElements(operation, element => {
       before(element, operation)
       operate(operation, () => {
-        const { name, args } = operation
-        if (!element[name]) {
-          console.warn(`CableReady callMethod failed due to missing '${name}' method for element:`, element)
-          return
+        const { element, receiver, method, args } = operation
+        const chain = method.split('.')
+        const obj = receiver === 'window' ? window : element
```

Comment:
Sadly, I don't think we can get away with a ternary here since we have no way to specify `document`.